tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -58,48 +67,63 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 					initializer = '''new «fixtureType»()'''
 				]
 			}
+			for (envParam : envParams) {
+				members += toField(envParam, envParam.name.envParamToVarName, typeRef(String)) [
+					initializer = '''System.getenv("«envParam.name»")'''
+				]
+			}
 			// Create test method
 			members += test.toMethod('execute', typeRef(Void.TYPE)) [
 				exceptions += typeRef(Exception)
 				annotations += annotationRef('org.junit.Test') // make sure that junit is in the classpath of the workspace containing the dsl
-				body = [test.generateMethodBody(trace(test, true))]
+				body = [test.generateMethodBody(trace(test, true), envParams)]
 			]
 		]
 
 	}
 
-	def void generateMethodBody(TestCase test, ITreeAppendable output) {
-		test.steps.forEach[generate(output.trace(it))]
+	def void generateMethodBody(TestCase test, ITreeAppendable output, Set<EnvParam> envParams) {
```

Comment:
Why convert the parameters to a `Set` here? Would be nice to have the generated checks in the order how they were specified in the TCL.